pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -94,19 +119,11 @@ def __enter__(self):
         print(msg)
         bokchoy_utils.start_servers(self.default_store)
 
-        msg = colorize('green', "Waiting for servers to start...")
-        print(msg)
-        bokchoy_utils.wait_for_test_servers()
-
-    def __exit__(self, exc_type, exc_value, traceback):
-        super(BokChoyTestSuite, self).__exit__(exc_type, exc_value, traceback)
-
-        msg = colorize('green', "Cleaning up databases...")
-        print(msg)
-
-        # Clean up data we created in the databases
-        sh("./manage.py lms --settings bok_choy flush --traceback --noinput")
-        bokchoy_utils.clear_mongo()
+    def interruptable_prompt(self):
+        """
+        Stops running process and waits for user input before continuing.
+        """
+        sh('read -p "Press enter to stop servers." PROMPTVAR')
```

Comment:
I'd prefer if you had to ctrl-C in order to stop the server, rather than any input. I think otherwise it is too easy to accidentally hit a key in the wrong terminal window.